tmva/sofie_parsers/src/RModelParser_ONNX.cxx

```diff
@@ -159,6 +159,43 @@ std::unique_ptr<ROperator> make_ROperator_Transpose(const onnx::NodeProto& nodep
    return op;
 }
 
+
+std::unique_ptr<ROperator> make_ROperator_Max(const onnx::NodeProto& nodeproto, const onnx::GraphProto& /*graphproto */, std::unordered_map<std::string, ETensorType>& tensor_type){
+
+   ETensorType input_type = ETensorType::UNDEFINED;
+
```

Comment:
Here it should not be 2 but use 
`nodeproto.input_size()` to retrieve all the input names